core/src/main/scala/kafka/log/Log.scala

```diff
@@ -133,6 +135,24 @@ class Log(val dir: File,
     },
     tags)
 
+  newGauge("LogSegments",
+    new Gauge[util.List[String]] {
+      def value = {
+        val list = logSegments.toSeq.map(
+          seg => s"baseOffset=${seg.baseOffset}, created=${seg.created}, logSize=${seg.size}, indexSize=${seg.index.sizeInBytes()}"
+        )
```

Comment:
Thanks for addressing the feedback. Very minor point, but the following matches our style convention:

```scala
logSegments.toSeq.map { seg =>
  s"baseOffset=${seg.baseOffset}, created=${seg.created}, logSize=${seg.size}, indexSize=${seg.index.sizeInBytes()}"
}